src/test/smart-components/platform/platform-inventories.test.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import configureStore from 'redux-mock-store' ;
+import thunk from 'redux-thunk';
+import { Provider } from 'react-redux';
+import promiseMiddleware from 'redux-promise-middleware';
+import { Route, MemoryRouter } from 'react-router-dom';
+import { notificationsMiddleware } from '@redhat-cloud-services/frontend-components-notifications/';
+import { mount, shallow } from 'enzyme';
+import { shallowToJson } from 'enzyme-to-json';
+import PlatformInventories from '../../../smart-components/platform/platform-inventories';
+import { TOPOLOGICAL_INVENTORY_API_BASE, SOURCES_API_BASE } from '../../../utilities/constants';
+import { FETCH_PLATFORM, FETCH_PLATFORM_INVENTORIES } from '../../../redux/action-types';
+import { platformInitialState } from '../../../redux/reducers/platform-reducer';
+import { act } from 'react-dom/test-utils';
+
+describe('<PlatformInventories />', () => {
+  let initialProps;
+  const middlewares = [ thunk, promiseMiddleware(), notificationsMiddleware() ];
+  let mockStore;
+  let initialState;
+
+  const ComponentWrapper = ({ store, initialEntries = [ '/foo' ], children }) => (
```

Comment:
A lot of tests use ComponentWrapper, I think it would be great to make a helper from this (but of course, it's not in the scope of this PR)

Reply:
agree

Reply:
agree